Sources/SwiftDocC/Model/Rendering/Symbol/PropertyListDetailsRenderSection.swift

```diff
@@ -33,41 +33,55 @@ public enum TitleStyle: String, Codable, Equatable {
 }
 
 /// A section that contains details about a property list key.
-struct PlistDetailsRenderSection: RenderSection, Equatable {
-    var kind: RenderSectionKind = .plistDetails
+public struct PropertyListDetailsRenderSection: RenderSection, Equatable {
+    public var kind: RenderSectionKind = .plistDetails
     /// A title for the section.
-    var title = "Details"
+    public var title = "Details"
     
     /// Details for a property list key.
-    struct Details: Codable, Equatable {
+    public struct Details: Codable, Equatable {
         /// The name of the key.
-        let rawKey: String
+        public let rawKey: String
         /// A list of types acceptable for this key's value.
-        let value: [TypeDetails]
+        public let value: [TypeDetails]
         /// A list of platforms to which this key applies.
-        let platforms: [String]
+        public let platforms: [String]
         /// An optional, human-friendly name of the key.
-        let displayName: String?
+        public let displayName: String?
         /// A title rendering style.
-        let titleStyle: PropertyListTitleStyle
-        
+        public let titleStyle: PropertyListTitleStyle
+
         enum CodingKeys: String, CodingKey {
             case rawKey = "name"
             case value
             case platforms
             case displayName = "ideTitle"
             case titleStyle
         }
+
+        public init(rawKey: String, value: [TypeDetails], platforms: [String], displayName: String?, titleStyle: PropertyListTitleStyle) {
+            self.rawKey = rawKey
+            self.value = value
+            self.platforms = platforms
+            self.displayName = displayName
+            self.titleStyle = titleStyle
+        }
     }
     
     /// The details of the property key.
-    let details: Details
+    public let details: Details
+
+    public init(kind: RenderSectionKind = .plistDetails, title: String = "Details", details: Details) {
+        self.kind = kind
```

Comment:
We don't want callers specifying a different kind for any of the render sections. That would result in unexpected JSON when DocC render uses the "kind" to determine what data to read.

(IIRC the reason why it's not `let` has something to do with decoding.)

```suggestion
    public init(title: String = "Details", details: Details) {
        self.kind = .plistDetails
```

Reply:
Good catch, thanks, I'm just going to delete this line because the ivar declaration specifies a default value already.